packages/python-packages/api-stub-generator/azure_sdk_pylintrc

```diff
@@ -0,0 +1,55 @@
+[MASTER]
```

Comment:
I opted not to do that because I didn't want someone to 1) need network connectivity to run APIVIew and 2) redownload the rcfile every time they run it. I figured the frequency of changes would be low enough that keeping them in sync and checking via CI would be more economical than making network calls every time (if only to verify that the local copy still matches).